tests/transactions/tests.py

```diff
@@ -443,8 +443,25 @@ def test_atomic_does_not_leak_savepoints_on_failure(self):
                 # This is expected to fail because the savepoint no longer exists.
                 connection.savepoint_rollback(sid)
 
-    @skipIf(connection.features.autocommits_when_autocommit_is_off,
-            "This test requires a non-autocommit mode that doesn't autocommit.")
+
+@skipIf(
+    connection.features.autocommits_when_autocommit_is_off,
+    "This test requires a non-autocommit mode that doesn't autocommit."
+)
+class NonAutocommitTests(TransactionTestCase):
+
+    available_apps = []
+
+    def test_orm_query_after_error(self):
+        """#27504 -- Cannot Make ORM Queries After an Error and Rollback In Non_autocommit Mode"""
```

Comment:
State the expected behavior in test docstrings and wrap docstrings at 79 chars per [Python style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style):
```
"""
ORM queries are allowed after an error and a rollback in non-autocommit
mode (#27504).
"""
```